src/legacy/core_plugins/management/public/np_ready/services/index_pattern_management/creation/config.ts

```diff
@@ -35,7 +35,7 @@ const indexPatternButtonDescription = i18n.translate(
 );
 
 export class IndexPatternCreationConfig {
-  public key = 'default';
+  public readonly key = 'default';
```

Comment:
In `ui/management`, this was `static key`. It is changed because TypeScript doesn't allow access to static variables from instances.